Documentation/devel/on-disk-format.md

```diff
@@ -0,0 +1,37 @@
+On disk format
+==============
+
+The data directory is normally `/var/lib/rkt` but rkt can be requested to use
+another directory via the `--dir` command line argument.
+
+#### CAS database
+
+The CAS database is stored in `/var/lib/rkt/cas/db`. The database schema can be
+migrated to newer versions (#706).
+
+#### CAS
+
+The CAS also uses other directories:
+
+- `/var/lib/rkt/cas/blob`
+- `/var/lib/rkt/cas/imagelocks`
+- `/var/lib/rkt/cas/imageManifest`
+- `/var/lib/rkt/cas/tree`
+- `/var/lib/rkt/cas/treestorelocks`
```

Comment:
I'd remove the listing. As long as stage1 does not have to know about it, it is none of it's business.